src/Tribe/Capabilities.php

```diff
@@ -15,24 +46,62 @@ class Tribe__Events__Capabilities {
 			'delete_private_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'author' => array( // full permissions for content the user created
+		],
+		'author' => [ // full permissions for content the user created
 			'read',
 			'edit_posts',
 			'edit_published_posts',
 			'delete_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'contributor' => array( // create, but not publish
+		],
+		'contributor' => [ // create, but not publish
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 39dcf8aabb010b1703f20a991c60c681</sub>